c_models/mcmc_cholesky/src/mcmc_cholesky.c

```diff
@@ -1,5 +1,5 @@
 #include "mcmc_cholesky.h"
-#include "../mcmc_models/mcmc_model.h"
+#include "../../mcmc_models/src/mcmc_model.h"
```

Comment:
The default compile will find files relative to a source folder using angle-brackets to avoid needing relative includes if you prefer...

Reply:
Done - I agree it's better to do it this way round.  Needed a bit of editing of the makefiles, but it works now.